configs/etc/ssh/sshd_config

```diff
@@ -0,0 +1,87 @@
+# Package generated configuration file
```

Comment:
shouldn't it belong to config-stretch only?

Reply:
it can be but it is actually the default config from wheezy. Should i move it to only stretch?

Reply:
I think so. No need to substitute the config with the same file